.github/workflows/master.yaml

```diff
@@ -0,0 +1,26 @@
+name: master
+on:
+  push:
+    branches:
+    - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: docker://l.gcr.io/google/bazel:2.1.0
+    steps:
+    - name: Check out source code
+      uses: actions/checkout@v1
+    - name: Buildifier
+      run: bazel run @com_github_bazelbuild_buildtools//:buildifier
```

Comment:
Could you sync this against latest bb-storage and enable `golint` while you're at it? Thanks!